src/context.rs

```diff
@@ -1,21 +1,22 @@
 use crate::ast::Value;
 use crate::schema::Schema;
-use fnv::FnvHashMap;
 use uuid::Uuid;
 
+type HashMap<K, V> = fnv::FnvHashMap<K, V>;
```

Comment:
Might be confused with `std::collections::HashMap`

Reply:
Yes, but I can not think about a new name? do you have any suggestion?

Reply:
rename to FastHashMap